Storage/Storage/CoreData/CoreDataIterativeMigrator.swift

```diff
@@ -144,20 +144,34 @@ private extension CoreDataIterativeMigrator {
         return tempDestinationURL
     }
 
-    /// Move the original source store to a backup location.
+    /// Move the original source store files to a backup location (typically /Documents/backup).
     ///
-    func makeBackup(at storeURL: URL) throws -> URL {
-        let backupURL = storeURL.deletingLastPathComponent().appendingPathComponent("backup")
-        try? fileManager.removeItem(at: backupURL)
-        try? fileManager.createDirectory(atPath: backupURL.path, withIntermediateDirectories: false, attributes: nil)
+    /// The files that will be moved are:
+    ///
+    /// - {store_filename}.sqlite
+    /// - {store_filename}.sqlite-wal
+    /// - {store_filename}.sqlite-shm
+    ///
+    /// Where {store_filename} is most probably "WooCommerce".
+    ///
+    func moveStoreFilesToBackupFolder(storeURL: URL) throws -> URL {
+        let storeFolderURL = storeURL.deletingLastPathComponent()
+        let backupURL = storeFolderURL.appendingPathComponent("backup")
+
+        if fileManager.fileExists(atPath: backupURL.path) {
+            try fileManager.removeItem(at: backupURL)
```

Comment:
I think this needs to be a `try?` so that if it fails it continues with the next statement.

Reply:
@astralbodies What Jaclyn said is correct. I forgot to highlight that the `FileManager` error is not very clear:

```
Error Domain=NSCocoaErrorDomain Code=4 "“WooCommerce.sqlite.~” 
couldn’t be moved to “backup” because either the former doesn’t exist, or the 
folder containing the latter doesn’t exist.
```

The problem can either be:

- The `backup` folder does not exist
- The `WooCommerce.sqlite.~` does not exist

Here, I'm intentionally making things fail if the `backup` folder fails to be recreated so that we'll more info if that is indeed the problem. 

Another thing we could do is to actually just not create a backup at all since we don't do anything with it. 😬 What do you think?

Reply:
@shiki Skipping the backup makes the most sense. The only reason we backed things up was because the original ALIterativeMigrator did it. We *never* wrote code to use that file. If migration fails, the db should just get whacked.

Reply:
As you wish! I shall happily delete that backup routine.

Reply:
note for future: maybe we can leave the more significant changes (like changing the migration flow) for later next time, since it's a hotfix and already tested :)